static/js/tinderMyType.js

```diff
@@ -0,0 +1,41 @@
+let readingListStorage = document.getElementById('readingListStorage');
```

Comment:
Since we're not using any build tools, I would avoid ES6 in front-end JS to improve browser compatibility.